src/plugins/vis_types/timeseries/public/application/components/lib/create_number_handler.ts

```diff
@@ -12,6 +12,12 @@ import { TimeseriesVisParams } from '../../../types';
 export const createNumberHandler = (
   handleChange: (partialModel: Partial<TimeseriesVisParams>) => void
 ) => {
-  return (name: keyof Metric, defaultValue?: string) => (e: React.ChangeEvent<HTMLInputElement>) =>
-    handleChange?.({ [name]: Number(e.target.value ?? defaultValue) });
```

Comment:
nit:
I think `e.target.value ? Number(e.target.value) : defaultValue` also should be fine. 
But I'm ok with your solution too

Reply:
From my perspective default value and possibility to clear value is different things. WDYT?